src/main/java/Entities/OrderItem.java

```diff
@@ -13,6 +13,10 @@ public OrderItem(ObjectId foodItemID, Food food, int numberOfItem) {
         this.numberOfItem = numberOfItem;
     }
 
+    public float getPrice() {
```

Comment:
Could create an interface OrderItem with getPrice etc. and implement different items such as GeneralOrderItem or TaxFreeItem so they have different implementation for different purposes